### PR TITLE
added new Json data

### PR DESCRIPTION
**Pull request by Ugoo-Eze**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1049"></a>

### CODE REVIEW

File: public/data/Ugoo-Eze.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "UGOO EZE",
+  "type": "personal",
+  "bio": "FULLSTACK WEB DEVELOPER.THANKS TO JAVASCRIPT!",
+  "avatar": "https://github.com/Ugoo-Eze.png",
+  "links": [
+    {
+      "name": "Check my Projects out on Github",
```
Comment (syedsohan) [inline public/data/Ugoo-Eze.json:8]:
```suggestion
      "name": "Check my Projects out on GitHub",
```

File: public/data/Ugoo-Eze.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "UGOO EZE",
+  "type": "personal",
+  "bio": "FULLSTACK WEB DEVELOPER.THANKS TO JAVASCRIPT!",
```
Comment (syedsohan) [inline public/data/Ugoo-Eze.json:4]:
```suggestion
  "bio": "Fullstack Web Developer. Thanks To JavaScript!",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]
Could you please revert the `package-lock.json` file?

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Please revert changes **`package-lock.json`** file as stated by **syedsohan**

File: package-lock.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.59.1",
+      "version": "0.62.2",
```
Comment (Panquesito7) [inline package-lock.json:9]:
```suggestion
      "version": "0.59.1",
```

File: public/data/Ugoo-Eze.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      "name": "Check out my journey on Twitter",
+      "url": "https://twitter.com/DerHerr66739164",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started my Tech Journey",
+      "date": "August 2020",
+      "icon": "Laptop",
```
Comment (Panquesito7) [inline public/data/Ugoo-Eze.json:22]: Are you sure there's a laptop icon? I think there's not. 😅

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Please remove the changes to package-lock.json. Also, the list of available icons is shown in the readme file.

File: public/data/Ugoo-Eze.json (hunk 4 above)
Comment (KendallDoesCoding) [inline public/data/Ugoo-Eze.json:22]:
```suggestion
```
For milestones, there's no need of any icon, and also as @Panquesito7 stated there's no laptop icon 😅

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: public/data/Ugoo-Eze.json (hunk 4 above)
Comment (KendallDoesCoding) [reply public/data/Ugoo-Eze.json:22]: Actually a icon is prefered/

Comment (Ugoo-Eze) [reply public/data/Ugoo-Eze.json]: sorry about that 😪

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM, Thank you for your contribution :rocket: :smile:

**Pull request merged**